### ISSUE TITLE
ERROR: No matching distribution found for torch==2.0.0+cu117

### ISSUE THREAD

**Issue by hesimin** [SCUTlihaoyu/open-chat-video-editor#15]
windows 10 GPU 安装报错：
pip install -r requirements.txt

Collecting toolz==0.12.0
  Using cached toolz-0.12.0-py3-none-any.whl (55 kB)
ERROR: Could not find a version that satisfies the requirement torch==2.0.0+cu117 (from versions: 1.7.1, 1.8.0, 1.8.1, 1.9.0, 1.9.1, 1.10.0, 1.10.1, 1.10.2, 1.11.0, 1.12.0, 1.12.1, 1.13.0, 1.13.1, 2.0.0, 2.0.1)
ERROR: No matching distribution found for torch==2.0.0+cu117

**Comment by moralibiao** [comment]
> windows 10 GPU 安装报错： pip install -r requirements.txt
> 
> Collecting toolz==0.12.0 Using cached toolz-0.12.0-py3-none-any.whl (55 kB) ERROR: Could not find a version that satisfies the requirement torch==2.0.0+cu117 (from versions: 1.7.1, 1.8.0, 1.8.1, 1.9.0, 1.9.1, 1.10.0, 1.10.1, 1.10.2, 1.11.0, 1.12.0, 1.12.1, 1.13.0, 1.13.1, 2.0.0, 2.0.1) ERROR: No matching distribution found for torch==2.0.0+cu117

I met this issue as well. What you need to do is following the recommendation like upgrading the version from  2.0.0+cu117 to 2.0.0 or 2.0.1. Please give it a try.

**Comment by hesimin** [comment]
> 

I tried to upgrade to version 2.0.0, but encountered a new issue with the version dependency error of Numpy.   😢

**Comment by hesimin** [comment]
> 

I tried to upgrade to version 2.0.0, but encountered a new issue with the version dependency error of Numpy. 😢

**Comment by moralibiao** [comment]
> I tried to upgrade to version 2.0.0, but encountered a new issue with the version dependency error of Numpy. 😢

Yes, It could be, you may manually fix the version issue caused by various modules one by one. Finally, you might meet the issue I reported before and it is still opened.

**Comment by linuxdevopscn** [comment]
根据你的cuda版本来安装对应的torch
https://pytorch.org/get-started/locally/
